clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -174,6 +174,27 @@ public class AzureComputeClient extends AzureBaseClient {
     )
   }
 
+  ServiceResponse<Void> disableServerGroup(String resourceGroupName, String serverGroupName) {
+    VirtualMachineScaleSetsOperations ops = getAzureOps(
+      client.&getVirtualMachineScaleSetsOperations, "Get operations object", "Failed to get operation object") as VirtualMachineScaleSetsOperations
+
+    List<String> instanceIds = this.getServerGroupInstances(resourceGroupName,serverGroupName)?.collect {it.resourceId}
+
+    ops.powerOff(resourceGroupName, serverGroupName, instanceIds)
+
+    // TODO: investigate if we can deallocate the VMs
+    //ops.deallocate(resourceGroupName, serverGroupName, instanceIds)
+  }
+
+  ServiceResponse<Void> enableServerGroup(String resourceGroupName, String serverGroupName) {
+    VirtualMachineScaleSetsOperations ops = getAzureOps(
+      client.&getVirtualMachineScaleSetsOperations, "Get operations object", "Failed to get operation object") as VirtualMachineScaleSetsOperations
```

Comment:
Why do we have retry logic around the "get operations" methods? These methods just instantiate and initialize the class that defines the available calls to Azure for that resource . There's no rest call actually being made. I thought the retry logic was for when we actually made a call to Azure